joomla/40-backend-template
# switcher as web component

Pull Request for Issue # .

### Summary of Changes
Switcher is now a true web component.

### Why?
- Truly decouple css of the component from any css framework
- Allow the component to disconnect/reconnect without breaking (eg try resizing the article edit and you'll figure out what's fixed there)
- support for web components is looking great now (Chrome, Opera, Safari, FF(next ver)) only MS is missing, but should come before J4
- Prepare for the future, eg support all the frameworks out there: check https://custom-elements-everywhere.com



### Testing Instructions
@ciar4n can you check that everything is ok here (eg edit an article and check that the resize is working ok now and also that save is not messed up)


### Expected result
Not broken


### Actual result
Broken


### Documentation Changes Required


@wilsonge you'll need to merge this
